production/tanka/grafana-agent-operator/internal/operator.libsonnet

```diff
@@ -0,0 +1,57 @@
+function(name='grafana-agent-operator', namespace='') {
+    local k = (import 'ksonnet-util/kausal.libsonnet') { _config+:: { namespace: namespace } },
+
+    local container = k.core.v1.container,
+    local policyRule = k.rbac.v1.policyRule,
+    local serviceAccount = k.core.v1.serviceAccount,
+    local deployment = k.apps.v1.deployment,
+
+    local this = self,
+
+    _images:: {
+      agent_operator: 'grafana/agent-operator:v0.26.0-rc.0',
```

Comment:
This could just be a parameter to the function. Then callers know how to change the image, and can store it in a top level config that get's passed down when instantiating the operator.